### DISCUSSION TITLE
Add support for continue and break statements

### DISCUSSION THREAD

**Discussion by nielstron** [OpShin/opshin#576] [Ideas / Improvements]
#297 introduced the option to implement `continue` and `break` statements. This did not exist in hebi before since it did not support loops.

One possible implementation is to keep track of a respective call required to return to the closest loop beginning or end directly (i.e. call `adjusted_next` or `while`/`for`)

> [Bug bounty](https://github.com/OpShin/opshin/blob/main/BUG_BOUNTY.md): 1000 ADA

**Comment by abhishek818** [comment]
@nielstron hey is the bug bounty still open? will have a attempt at this.

**Comment by nielstron** [comment]
Hi @abhishek818, yes it's still open :)
